_includes/post-meta.html

```diff
@@ -2,10 +2,18 @@
   <div class="post-meta">
     <h4>About the author</h4>
     {% for person in site.team %}{% if person.name == page.author %}
-      <a href="{{ page.author | downcase | replace: ' ', '-'  | prepend: "/team/" | strip_newlines }}">
+
+      {% comment %} Display depends on author's employment status {% endcomment %}
+      {% if person.status == "former" %}
         {% picture team_post_meta {{ person.photo }} alt="{{ person.name }}" %}
-      </a>
-      <p class="post-author-desc">{{ person.job_description }}</p>
+        <p class="post-author-desc"> {{ person.name }} is a former Savas Labs employee.</p>
```

Comment:
Mega nit-pick! Please remove the space after the `>` and before `{{ person.name}}`